asapdiscovery/simulation/scripts/MproP0009.py

```diff
@@ -0,0 +1,182 @@
+# an openforcefield Molecule object that was created from a RDKit molecule.
+# The topology part is described in the openforcefield API but the positions part grabs the first (and only)
+# conformer and passes it to Modeller. It works. Don't ask why!
+modeller.add(ligand_mol.to_topology().to_openmm(), ligand_mol.conformers[0].to_openmm())
+
+
+# We need to temporarily create a Context in order to identify molecules for adding virtual bonds
+integrator = openmm.VerletIntegrator(1 * unit.femtoseconds)
+system = system_generator.create_system(modeller.topology, molecules=ligand_mol)
+context = openmm.Context(system, integrator, openmm.Platform.getPlatformByName('CUDA'))
+molecules_atom_indices = context.getMolecules()
+del context, integrator, system
+
+system = system_generator.create_system(modeller.topology, molecules=ligand_mol)
+
+custom_bond_force = openmm.CustomBondForce('0')
+for molecule_index in range(len(molecules_atom_indices)-1):
+    custom_bond_force.addBond(molecules_atom_indices[molecule_index][0], molecules_atom_indices[molecule_index+1][0], [])
+system.addForce(custom_bond_force)
+
+# Input Files
+
+from openmm.app import ForceField
+#pdb = PDBFile(args.input_pdb_path)
+
+# System Configuration
+
+from openmm.app import PME, HBonds
+
+nonbondedMethod = PME
+nonbondedCutoff = 1.0*nanometers
+ewaldErrorTolerance = 0.000001
+constraints = HBonds
+rigidWater = True
+constraintTolerance = 0.000001
+hydrogenMass = 4.0*amu
+
+
+
+# Integration Options
+
+dt = 4.0 * unit.femtoseconds
+temperature = 300 * unit.kelvin
+friction = 1.0 / unit.femtosecond
+pressure = 1.0 * atmospheres
+barostatInterval = 25
+
+from openmm.app import PDBReporter
+from openmm.app import DCDReporter
+from openmm.app import StateDataReporter
+from openmm.app import CheckpointReporter
+from openmm import Platform
+# Simulation Options
+
+steps = 10000
+equilibrationSteps = 100
+platform = Platform.getPlatformByName('CUDA')
+dcdReporter = DCDReporter(os.path.join(args.output_dir,'trajectory.dcd'), 50)
+dataReporter = StateDataReporter(os.path.join(args.output_dir,'log.txt'), 50, totalSteps=steps,
+    step=True, speed=True, progress=True, potentialEnergy=True, temperature=True, separator='\t')
+checkpointReporter = CheckpointReporter(os.path.join(args.output_dir,'checkpoint.chk'), 50)
+
+# Prepare the Simulation
+from openmm import MonteCarloBarostat
+from openmm import LangevinMiddleIntegrator
+from openmm.app import *
+print('Building system...')
+#topology = pdb.topology
+#positions = pdb.positions
+#modeller = Modeller(pdb.topology, pdb.positions)
+modeller.addSolvent(system_generator.forcefield, padding=0.9*unit.nanometers, model='tip3p')
+system = forcefield.createSystem(modeller.topology, nonbondedMethod=nonbondedMethod, nonbondedCutoff=nonbondedCutoff,
+    constraints=constraints, rigidWater=rigidWater, ewaldErrorTolerance=ewaldErrorTolerance, hydrogenMass=hydrogenMass)
```

Comment:
@kendallrlemons this overwrites the system created on line 92, which loses the virtual bond!